v2.0/src/Components/Pages/ActionsPage/ActionsPage.js

```diff
@@ -71,14 +71,19 @@ class ActionsPage extends React.Component {
   findCommon() {
     const actions = this.props.actions;
     const values = this.state.check_values ? this.state.check_values : [];
+    if (values.length == 0)
+      return null;
+
     const common = [];
     if (actions) {
       for (let i = 0; i < actions.length; i++) {
         const action = actions[i];
-        const tag = action.tags[i];
-        if (tag) {
-          if (values.includes(tag.id) && !common.includes(action)) {
-            common.push(action);
+        for (let j=0; j< action.tags.length; j++) {
```

Comment:
Nice work.  One very nit comment:
`action.tags.length` assumes the following:
1.  `action` always exists
2. `action.tags` always exists and is a list 

 In actual fact, we are not guaranteed any of those which makes it a bit dangerous.

We could make a variable prior like
`const actionTags = (action && action.tags) || []` 
then we can do `actionTags.length` 
This way we are making sure we always have an action and action.tags and worst case scenario we have an empty list